packages/terra-menu/src/_MenuUtils.js

```diff
@@ -15,8 +17,27 @@ const isFullScreen = (isHeightBounded, isWidthBounded, boundingFrame, popupWidth
   return isHeightBounded && (width >= maxWidth || isWidthBounded);
 };
 
+const isMac = () => navigator.userAgent.indexOf('Mac') !== -1 && navigator.userAgent.indexOf('Win') === -1;
+
+const totalItems = (children) => {
+  let count = 0;
+  React.Children.map(children, (item) => {
+    if (item.props.text) {
+      count += 1;
+    } else if (item.props.children) {
+      React.Children.forEach(item.props.children, () => {
```

Comment:
do we need count dividers also ?

Reply:
No, considering only menu items (disabled/enabled) and items under group